ui/src/reducers/index.js

```diff
@@ -54,6 +58,32 @@ const sortRows = (rows, index, direction) => {
   return sortedRows;
 };
 
+/**
+ * Check if all of the table column values are the same
+ *
+ * @param {Array} rows The table rows in which we will compare values
```

Comment:
@craicoverflow should we be doing this in other methods?

Reply:
I was wondering that also and I think we should as it the standard JavaScript way.

Look how clear it makes a function when you hover over it - you know exactly what it does and what the params are for. Documentation can also be generated when this format is used.

![Screenshot from 2019-03-19 10-40-28](https://user-images.githubusercontent.com/11743717/54599961-a693c380-4a33-11e9-929b-3ebc1a6ee86b.png)

Ref: [JSDoc](http://usejsdoc.org/)